source/StaticBillboardSet.cpp

```diff
@@ -705,8 +705,11 @@ MaterialPtr StaticBillboardSet::getFadeMaterial(const Ogre::MaterialPtr &protoMa
             GpuProgramParametersSharedPtr params = pass->getVertexProgramParameters();
 
             //glsl can use the built in gl_ModelViewProjectionMatrix
-            if (!isglsl)
-               params->setNamedAutoConstant("worldViewProj", GpuProgramParameters::ACT_WORLDVIEWPROJ_MATRIX);
+			if (!isglsl)
+			{
+				static const Ogre::String worldViewProj = "worldViewProj";

```

Comment:
seems like an unrelated/ unnecessary change

Reply:
I was getting an error running the samples that worldViewProj wasn't a valid parameter.  I read that this is due to compiler optimization removing the "unused" worldViewProj.  This change allowed the plugin to run.